text-utilities/utilities.css

```diff
@@ -0,0 +1,56 @@
+.text-heading{
+    display: flex;
+    justify-content: center;
+    align-content: center;
+}
+
+.types-of-headings{
+    flex-direction: row;
+    text-align: center;
+    width: 30rem;
+}
+
+#small-heading{
+   
+    padding: 0.5rem;
+    margin: 1rem;
+    font-size: 20px;
+    font-weight: 500;
+}
+
+#medium-heading{
+    
+    padding: 0.5rem;
+    margin: 1rem;
+    font-size: 24px;
+    font-weight: 700;
+}
+
+#large-heading{
+    padding: 0.5rem;
+    margin: 1rem;
+    font-size: 28px;
+    font-weight: 800;
+}
+
+.type-text-container{
+    font-size: 18px;
+    width: max-content;
+}
+
+.para-type{
+    padding: 00.5rem;
+    margin: 0.5rem;
+}
+
+.para-type:nth-child(1){
+    font-weight: 600;
+}
+
+.para-type:nth-child(2){
+    font-weight: 500;
+}
+
+.para-type:nth-child(3){
+    color: gray;
+}
```

Comment:
Suggestion: Using nth child will not be appropriate in this case as you might not be able to use these utilities individually. In this container only 3rd child will be the gray text and if you only want one element with gray text then there will an issue.